gallery/src/components/NxDrawer/NxDrawerDisabledFunctionalityExample.tsx

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+
+import { NxDrawer,
+  NxButton,
+  NxP,
+  NxBackButton,
+  NxPageMain,
+  NxPageTitle,
+  NxH1,
+  useToggle
+} from '@sonatype/react-shared-components';
+
+export default function NxDrawerExample() {
+  const [showDrawerWithDisabledFunctionality, toggleDrawerWithDisabledFunctionality] = useToggle(false);
+  const [disabled, toggleDisabled] = useToggle(true);
+
+  return (
+    <>
+      <header className="nx-global-header">
+        <NxBackButton href="#/pages/Drawer" targetPageTitle="Documentation" />
+      </header>
+      <NxDrawer id="nx-drawer-disabled-functionality"
+                open={showDrawerWithDisabledFunctionality}
+                onClose={toggleDrawerWithDisabledFunctionality}
+                aria-labelledby="simple-drawer-disabled-title"
+                closeBtnDisabled={disabled}
+                closeBtnTooltip={disabled ? "Please save or revert changes" : "Close"}
+                >
+        <NxDrawer.Header>
+          <NxDrawer.HeaderTitle id="simple-drawer-disabled-title">
+            A Simple Drawer With Disabled Functionality
+          </NxDrawer.HeaderTitle>
+        </NxDrawer.Header>
+        <NxDrawer.Content tabIndex={0}>
```

Comment:
Not needed since the drawer contains a focusable item
```suggestion
        <NxDrawer.Content>
```

Reply:
https://github.com/sonatype/sonatype-react-shared-components/pull/1169/commits/32ff2199613674e20bc98f630c4d01e6b8f83825